### DISCUSSION TITLE
ALEO CONTRIBUTOR BADGE

### DISCUSSION THREAD

**Discussion by vheeq1** [community/community#103247] [GitHub Education]
### Hi everyone,

This is wickedness and very unfair!!!! Was among the early people to submit and still didn’t get the Aleo contributor badge!

**Comment by gil-son** [comment]
Hello

Look at that documentation about the [rules](https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.galaxy-brain.md)

In resume: You need two submitted answers in any categories you want to get the Galaxy Brain achievement

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @vheeq1. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by alsheroziy** [comment]
Hello everyone, I just joined the github community, please tell me the rules
